mmdet/models/dense_heads/ddod_head.py

```diff
@@ -0,0 +1,604 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, Scale, bias_init_with_prob, normal_init
+from mmcv.runner import force_fp32
+
+from mmdet.core import (anchor_inside_flags, build_assigner, build_sampler,
+                        images_to_levels, multi_apply, reduce_mean, unmap)
+from mmdet.core.bbox import bbox_overlaps
+from ..builder import HEADS, build_loss
+from .anchor_head import AnchorHead
+
+
+@HEADS.register_module()
+class DDODHead(AnchorHead):
```

Comment:
add unit test for `DDODHead`, refer to https://github.com/open-mmlab/mmdetection/blob/6ddbff8406aec8b414462badbd57c4013b594afe/tests/test_models/test_dense_heads/test_atss_head.py#L8

add ddod detector, refer to https://github.com/open-mmlab/mmdetection/blob/6ddbff8406aec8b414462badbd57c4013b594afe/mmdet/models/detectors/atss.py#L7